reference/events.rst

```diff
@@ -0,0 +1,189 @@
+Symfony Framework Events
+========================
+
+Kernel Events
+-------------
+
+Each event thrown by the Kernel is a subclass of
+:class:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent`. This means that
+each event has access to the following information:
+
+:method:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent::getRequestType`
+    Returns the *type* of the request (``HttpKernelInterface::MASTER_REQUEST`` or
+    ``HttpKernelInterface::SUB_REQUEST``).
+
+:method:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent::getKernel`
+    Returns the Kernel handling the request.
+
+:method:`Symfony\\Component\\HttpKernel\\Event\\KernelEvent::getRequest`
+    Returns the current ``Request`` being handled.
+
+.. _kernel-core-request:
+
+``kernel.request`` Event
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Event Class*: :class:`Symfony\\Component\\HttpKernel\\Event\\GetResponseEvent`
```

Comment:
please use `**` to make it strong, which is consistent with other reference detail listings.